tools/windows-node-installer/go.mod

```diff
@@ -18,26 +12,14 @@ require (
 	github.com/coreos/etcd v3.3.10+incompatible
 	github.com/coreos/go-systemd v0.0.0-20190321100706-95778dfbb74e // indirect
 	github.com/coreos/pkg v0.0.0-20180928190104-399ea9e2e55f // indirect
-	github.com/gogo/protobuf v1.2.1 // indirect
-	github.com/google/gofuzz v0.0.0-20170612174753-24818f796faf // indirect
-	github.com/googleapis/gnostic v0.3.1 // indirect
 	github.com/imdario/mergo v0.3.7 // indirect
-	github.com/json-iterator/go v1.1.5 // indirect
 	github.com/masterzen/winrm v0.0.0-20190308153735-1d17eaf15943
-	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
-	github.com/modern-go/reflect2 v0.0.0-20180701023420-4b7aa43c6742 // indirect
-	github.com/openshift/api v3.9.1-0.20190814194116-a94e914914f4+incompatible
-	github.com/openshift/client-go v0.0.0-20190813201236-5a5508328169
+	github.com/openshift/api v0.0.0-20200205145930-e9d93e317dd1
+	github.com/openshift/client-go v0.0.0-20191125132246-f6563a70e19a
 	github.com/spf13/cobra v0.0.5
-	github.com/stretchr/testify v1.3.0
-	golang.org/x/net v0.0.0-20190522155817-f3200d17e092 // indirect
-	golang.org/x/oauth2 v0.0.0-20180821212333-d2e6202438be // indirect
-	golang.org/x/time v0.0.0-20190308202827-9d24e82272b4 // indirect
-	google.golang.org/appengine v1.1.0 // indirect
-	gopkg.in/inf.v0 v0.9.1 // indirect
-	k8s.io/api v0.0.0-20190409021203-6e4e0e4f393b // indirect
-	k8s.io/apimachinery v0.0.0-20190404173353-6a84e37a896d
-	k8s.io/client-go v11.0.1-0.20190409021438-1a26190bd76a+incompatible
-	k8s.io/utils v0.0.0-20190809000727-6c36bc71fc4a // indirect
-	sigs.k8s.io/yaml v1.1.0 // indirect
+	github.com/stretchr/testify v1.4.0
+	golang.org/x/crypto v0.0.0-20191011191535-87dc89f01550 // indirect
+	golang.org/x/net v0.0.0-20191209160850-c0dbc17a3553 // indirect
+	k8s.io/apimachinery v0.16.7
+	k8s.io/client-go v0.16.7
```

Comment:
This and whatever else you want to pin should be under `replace` otherwise the next `go mod tidy` might whack it.

Reply:
That shouldn't be an issue anymore.  I believe we were using `replace`  as a hack to prevent that as we were importing non go mod libraries (nes with `+incompatible). I've ran `go mod tidy` multiple times after manually setting the version to an earlier version in the go.mod file, and there has been no problem.